src/Microsoft.Framework.ApplicationHost2/Program.cs

```diff
@@ -4,6 +4,10 @@
 using System;
 using System.Threading.Tasks;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Weird that it isn't the default... I don't think I changed it. Anyway, fixed settings and file.